src/west/commands/configuration.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
```

Comment:
I agree with you, but reason I named it like this, is because there is already a  `config.py` in `west` and I really dislike having multiple python files with same name.

Suggestion, rename `configuration.py` to `config.py` and the `Class` to conform to normal, but then instead rename existing `config.py` to `westconfig.py` and its internal `config` variable to `westconfig`.
Thoughts ?

Reply:
The import would be `west.westconfig` then, which is a bit repetitive... what about `conf.py`?

Reply:
how about swapping the names ?
So that `configuration.py --> config.py` as the command.
And `config.py --> configuration.py` then you'll have: `west.configuration` and `configuration.get`
(although `configuration` is a bit long, then I feel `conf.py` and `config.py` are too close)

If you still prefer `conf.py` over `configuration.py` that's also ok, as long as names differs ;)

Reply:
Sure, swapping sounds fine!

Reply:
Done